ui-v2/src/components/ui/docs-link.tsx

```diff
@@ -5,6 +5,8 @@ import { Icon } from "./icons";
 import { Button } from "./button";
 
 const DOCS_LINKS = {
+	"automations-guide":
+		"https://docs.prefect.io/v3/automate/index#automate-overview",
```

Comment:
I think we want this more specific page instead:
```suggestion
	"automations-guide": 	"https://docs.prefect.io/v3/automate/events/automations-triggers",
```

Reply:
thanks! updated in latest push